### DISCUSSION TITLE
Multiprotocol v1.3.2.77

### DISCUSSION THREAD

**Discussion by pascallanger** [pascallanger/DIY-Multiprotocol-TX-Module#567] [General]
<img align="center" width=300 src="https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/docs/images/multi.png" />
<img src=https://shields.io/github/downloads/pascallanger/DIY-Multiprotocol-TX-Module/v1.3.2.77/total.svg>

## Change Log since v1.3.2.61
- [DSM/DSMR](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#sub_protocol-dsmr_1f---5)
  - New DSMR subprotocol for DSMR surface receivers
  - Only 22 IDs available, use RX num to cycle through them
  - Telemetry enabled
- [Pelikan/SCX24](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#sub_protocol-scx24---2)
  - New SCX24 subprotocol for 3 channels surface receivers
  - TX: Axial AX-4 2.4GHz transmitter and Panda Hobby 3CH Smart Radio 2.4GHz (MT-305A) and Models: Axial SCX24: Deadbolt, Jeep Wranger Rubicon, Chevrolet 1967 C10, B-17 Betty and Panda Hobby: Tetra K1, X1, X2
- [OMP](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#OMP---77)
  - The OMP protocol now supports telemetry using the CC2500 (was using the NRF24L01 previously)
  - If the CC2500 RF component is not installed then the NRF24L01 will be used including telemetry.
- [MT99XX/Dragon](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#sub_protocol-dragon---6)
  - Added telemetry: A1 = battery voltage with a Ratio of 25.5, A2=battery low flag (0=off,>0=on) and RSSI = dummy value of 100
- [DSM](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#DSM---6)
  - Added a table detailling the different RX output ranges based on the radio settings
- [E016H](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#e016h---85)
  - Separated protocol from E01X due to the different structure
- [Hitec](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#HITEC---39)
  - Fixed RF tunning
- [Kyosho/FHSS](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#kyosho---73)
  - Fixed protocol after the introduction of sub protocol Hype
- [Devo](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#devo---7)
  - Fixed GPS year
- [FrSkyX2](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#FRSKYX2---64)
  - Updated bind to work with some new RXs code
- [CABELL](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#cabell---34)
  - Protocol optimization: 1.5KB saved, thanks Mike!
- Multi core
  - XN297 emulation layer changed
    - Loads of protocols have been touched by this change. Some testing has been done but please test on all your models
    - The XN297 emulation selects in this order:
      - the CC2500 if it is available and bitrate=250K. Configure the option field automatically for RF tune
      - the NRF for all bitrates if it is available
      - if NRF is not available and bitrate=1M then an invalid protocol is sent automatically to the radio
    - CC2500 with a 250K bitrate can now receive normal and enhanced payloads
    - CC2500 only modules have more protocols/subprotocols available: MJXQ, MT99XX, Q303 and XK
  - Packed protocols defintion: 400 bytes saved

Flashing these firmwares **require [Flash Multi v0.6.1](https://github.com/benlye/flash-multi/releases/tag/0.6.1) or newer**, check the [Firmware updates manual](https://www.multi-module.org/using-the-module/firmware-updates) and [troubleshooting section](https://github.com/benlye/flash-multi/blob/master/doc/Troubleshooting.md) if you have issues.

<hr /><em>This discussion was created from the release <a href='https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/releases/tag/v1.3.2.77'>Multiprotocol v1.3.2.77</a>.</em>

**Comment by DarkSoulRL** [comment]
Why is it possible to flash only through Flash Multi, and not through OpenTX?

**Reply by benlye** [reply]
It means if you use Flash Multi you must use that version (or newer).

You can still use OpenTX instead of Flash Multi.
